File: Lab7Calificado/Lab7_2025_2/CinestarSolution/CinestarBusiness/BO/IBaseBO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CinestarBusiness.BO
{
    public interface IBaseBO<T>
    {
        int Insertar(T objeto);
        int Modificar(T objeto);
        int Eliminar(int idObjeto);
        List<T> ListarTodos();
        T BuscarPorId(int id);
    }
}

File: Lab9Calificado/Lab11_SOAP_P3_RENIEC/ReniecSoft/ReniecSoftBusiness/src/main/java/pe/edu/pucp/reniecsoft/business/bo/IBaseBO.java
package pe.edu.pucp.reniecsoft.business.bo;

import java.util.List;

public interface IBaseBO <T>{
    int insertar(T objeto) throws Exception;
    int modificar(T objeto) throws Exception;
    int eliminar(String dni) throws Exception;
    List<T> listarTodos(T objeto) throws Exception;
    T buscarPorDni(String dni) throws Exception;
}
